exercises/two-fer/two_fer.go

```diff
@@ -6,7 +6,7 @@
 package twofer
 
 // ShareWith should have a comment documenting it.
-func ShareWith(string) string {
+func ShareWith(name string) string {
```

Comment:
This stub is intentionally incomplete.  But I think it is fine to add the name to make the func signature complete.